### PR TITLE
Index rename to indice

### PR DESCRIPTION
**Pull request by monsieurleberre**
Hopefully I didn't break anything, tried MarketData server and IndiceManager client, they look OK, and all tests are OK too 🤞

### CODE REVIEW

File: src/Trakx.Common/Core/IndiceComposition.cs [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+using Trakx.Common.Interfaces.Indice;
 
 namespace Trakx.Common.Core
 {
     /// <inheritdoc />
-    public class IndexComposition : IIndexComposition
+    public class IndiceComposition : IIndiceComposition
     {
-        //public IndexComposition() { }
+        //public IndiceComposition() { }
```
[the remark this answers is not in the document]
Comment (monsieurleberre) [reply src/Trakx.Common/Core/IndiceComposition.cs:11]:
```suggestion
```

File: src/Trakx.MarketData.Server/Models/IndicePricedModel.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        public static IndicePricedModel FromIndiceValuations(IIndiceValuation initialValuation,
+            IIndiceValuation currentValuation)
+        {
+            Guard.Against.Default(initialValuation, nameof(initialValuation));
+            Guard.Against.Default(currentValuation, nameof(currentValuation));
+
+            Debug.Assert(ValuationsAreInTheSameCurrency(initialValuation, currentValuation),
+                "All underlying component valuations should be quoted in the same currency.");
+
+            var indicePriced = new IndicePricedModel()
```
[the remark this answers is not in the document]
Comment (monsieurleberre) [reply src/Trakx.MarketData.Server/Models/IndicePricedModel.cs:31]:
```suggestion
            var indicePriced = new IndicePricedModel
```

File: src/Trakx.Persistence/IndiceDataProvider.cs [hunk 3]
```diff
@@ -7,58 +7,58 @@
...
-                var cacheKey = $"{indexSymbol}|asOf|{asOfUtc:yyMMddHHmmss}";
+                var cacheKey = $"{indiceSymbol}|asOf|{asOfUtc:yyMMddHHmmss}";
                 var def = await _memoryCache.GetOrCreateAsync(cacheKey, async entry =>
                 {
                     entry.SetSlidingExpiration(TimeSpan.FromSeconds(10));
                     
-                    var version = await GetVersionAtDate(indexSymbol, asOfUtc, cancellationToken);
-                    var composition = await RetrieveFullComposition(indexSymbol, version.Value, cancellationToken);
+                    var version = await GetVersionAtDate(indiceSymbol, asOfUtc, cancellationToken);
+                    var composition = await RetrieveFullComposition(indiceSymbol, version.Value, cancellationToken);
```
[the remark this answers is not in the document]
Comment (monsieurleberre) [reply src/Trakx.Persistence/IndiceDataProvider.cs:61]:
```suggestion
                    var composition = await RetrieveFullComposition(indiceSymbol, version.Value, cancellationToken).ConfigureAwait(false);
```

File: src/Trakx.MarketData.Server/Pages/_Host.cshtml [hunk 4]
```diff
@@ -19,6 +19,6 @@
     </app>
 
     <script src="_framework/blazor.server.js"></script>
-    <script src="npm-libs/public/index.js"></script>
+    <script src="npm-libs/public/indice.js"></script>
```
Comment (AymericNoel) [inline src/Trakx.MarketData.Server/Pages/_Host.cshtml:22]: Index here isn't the main page in html ?

File: src/Trakx.MarketData.Server/Startup.cs [hunk 5]
```diff
@@ -109,7 +109,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseSwaggerUI(c =>
             {
                 c.SwaggerEndpoint($"/swagger/{ApiVersion}/swagger.json", ApiName);
-                c.InjectJavascript($"public/index.js");
+                c.InjectJavascript($"public/indice.js");
```
Comment (AymericNoel) [inline src/Trakx.MarketData.Server/Startup.cs:112]: Same here ?

File: README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@
 # Trakx Tools
 A solution grouping several tools used at Trakx for creation, rebalancing, and pricing of indices. This workspace currently hosts a couple of different but related tools being developed at Trakx.
 - Trakx.Persistence relies on SQL Server to Entity to define and interact with a database where all indices are defined, and valuation will eventually be maintained.
-- Trakx.MarketData.Server is an Asp.Net Core server currently used by the exchange to retrieve details about the indexes, such as composition, and live Net Asset Value.
+- Trakx.MarketData.Server is an Asp.Net Core server currently used by the exchange to retrieve details about the indicees, such as composition, and live Net Asset Value.
```
Comment (AymericNoel) [inline README.md:11]:
```suggestion
- Trakx.MarketData.Server is an Asp.Net Core server currently used by the exchange to retrieve details about the indices, such as composition, and live Net Asset Value.
```

File: src/Trakx.MarketData.Server/Pages/_Host.cshtml (hunk 4 above)
Comment (monsieurleberre) [reply src/Trakx.MarketData.Server/Pages/_Host.cshtml:22]:
```suggestion
    <script src="npm-libs/public/index.js"></script>
```

File: src/Trakx.MarketData.Server/Startup.cs (hunk 5 above)
Comment (monsieurleberre) [reply src/Trakx.MarketData.Server/Startup.cs:112]:
```suggestion
                c.InjectJavascript($"public/index.js");
```

**Pull request merged**